test/contract-proxy-kit.ts

```diff
@@ -4,12 +4,11 @@ import { ethers as ethersMaj5 } from 'ethers-5';
 import Web3Maj1Min2 from 'web3-1-2';
 import Web3Maj2Alpha from 'web3-2-alpha';
 import { SafeInfo } from '@gnosis.pm/safe-apps-sdk';
-import CPK from '../src';
+import CPK, { SafeRelayTransactionManager } from '../src';
 import { zeroAddress } from '../src/utils/constants';
 import { Address } from '../src/utils/basicTypes';
 import Web3Adapter from '../src/ethLibAdapters/Web3Adapter';
 import { TransactionManagerNames } from '../src/transactionManagers/TransactionManager';
```

Comment:
Could also import these symbols from '../src' now possibly?

Also maybe should also expose TransactionManagerNames